test/integration/connection_settings_test.py

```diff
@@ -0,0 +1,105 @@
+import pytest
+import pyexasol
+from inspect import cleandoc
+
+
+@pytest.fixture
+def connection_with_quote_indent(dsn, user, password, schema):
+    yield pyexasol.connect(
+        dsn=dsn, user=user, password=password, schema=schema, quote_ident=True
+    )
```

Comment:
Do you need to close this afterwards?
```suggestion
    connection = pyexasol.connect(
        dsn=dsn, user=user, password=password, schema=schema, quote_ident=True
    )
    yield connection
    connection.close()
```

Reply:
Additionally: would it be possible to
* turn off autocommit
* remove all commits
* remove cleanup (`DROP TABLE`)
* rollback before closing the connection